fs/zbd_zenfs.cc

```diff
@@ -467,13 +477,23 @@ void ZonedBlockDevice::ResetUnusedIOZones() {
   }
 }
 
-Zone *ZonedBlockDevice::AllocateZone(Env::WriteLifeTimeHint file_lifetime) {
+IOStatus ZonedBlockDevice::AllocateZone(Env::WriteLifeTimeHint file_lifetime,
+                                        Zone **out_zone) {
```

Comment:
"out_zone" -> allocated_zone

Reply:
Prefixing pointer arguments used for returning allocated items with `out_` conveys intention to the reader. How about `out_allocated_zone` ?

Reply:
that would be fine